examples/js/controls/TransformControls.js

```diff
@@ -903,7 +913,7 @@
 
 		function onPointerMove( event ) {
 
-			if ( scope.object === undefined || scope.axis === null || _dragging === false || ( event.button !== undefined && event.button !== 0 ) ) return;
```

Comment:
yeah that is a good point you have.
But we have to do something about this expression, because:
When I open my project on the Edge Browser and wanna jump into the pointermove event (while the primary mouse button is down) I get 'event.button = -1' (sorry my previous post seems to be a bit false^^) but if we check for 'event.button !== 0' then I will always return and won't be able to move on the edge browser. Even though I can't image why I get -1...